Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,66 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-562-g3dbc9ff-src.tar.gz"
+  version "1.6.7-562"
+  sha256 "6aded257e11fa114dc21e8493ff96dfe3f8a692d62e096afb5105cecc179211d"
+  head "https://github.com/tarantool/tarantool.git", :shallow => false
```

Comment:
It wasn't the last time I tried -https://github.com/Homebrew/homebrew/pull/47495#discussion_r48695165